proto/Cargo.toml

```diff
@@ -27,11 +27,15 @@ prost-types = { version = "0.9", default-features = false }
 bytes = { version = "1.1", default-features = false }
 tonic = { version = "0.6", optional = true, default-features = false }
 serde = { version = "1.0", default-features = false }
+schemars = { version = "0.8", optional = true }
+base64 = { version = "0.13", optional = true }
 
 [dependencies.tendermint-proto]
 version = "=0.23.5"
 default-features = false
 
 [features]
-default = ["std"]
-std = ["tonic", "tonic/codegen", "tonic/transport", "tonic/prost"]
+default = ["std", "client"]
+client = ["std", "tonic", "tonic/codegen", "tonic/transport", "tonic/prost"]
+json-schema = ["std", "schemars"]
+std = ["base64"]
```

Comment:
Shouldn't this be the opposite?
```
base64 = ["std"]
```

Reply:
Here `std` is the feature and `base64` the optional crate that is enabled when the `std` feature is enabled. That said, I agree with your comment below (https://github.com/informalsystems/ibc-rs/pull/1914#discussion_r814571968) and will enable the base64 encoding (and therefore Go compat) for both std and no_std.